src/Microsoft.AspNet.Razor/Generator/Compiler/CodeTree/Chunks/TagHelpers/TagHelperChunk.cs

```diff
@@ -29,5 +52,10 @@ public class TagHelperChunk : ChunkBlock
         /// The HTML tag name.
         /// </summary>
         public string TagName { get; set; }
+
+        /// <summary>
+        /// Indicates whether or not the tag of the tag helpers HTML element is self-closing.
```

Comment:
- always "Gets" or "Gets or sets" even though existing Razor code is not-so-great about this
- use the same wording for this property as in `TagHelperExecutionContext`